modules/ROOT/pages/constraints/syntax.adoc

```diff
@@ -26,6 +26,11 @@ It may still throw an error if conflicting data, indexes, or constraints exist.
 Examples of this are nodes with missing properties, indexes with the same name, or constraints with same schema but a different conflicting constraint type.
 An informational notification is returned in case nothing happens showing the existing constraint which blocks the creation.
 
+=== Syntax [grammar]
+:syntax-filename: constraints/create
+
+include::partial$syntax.adoc[]
```

Comment:
previously we had small syntax bits per constraint type, now we have one larger section covering all of them which might be less nice?